bosh-setup/azuredeploy.json

```diff
@@ -113,10 +113,10 @@
 
     "baseUriAzureCloud": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/bosh-setup/",
```

Comment:
Why are we hardcoding this instead of using the best practice from here: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc